### DISCUSSION TITLE
verse_onprem/3.2/admin/vop_configuring_server

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#163] [Q&A]
# verse_onprem/3.2/admin/vop_configuring_server

To install and configure HCL Verse on a Domino mail server, complete the following steps.

https://help.hcl-software.com/verse_onprem/3.2/admin/vop_configuring_server.html

**Comment by kbmsg** [comment]
Step 10 probably should say to extract to domino\data.
But then what is the story with the "Servers-lookup-<version>.jar" file?
I presume it goes to <Domino data directory>/domino/workspace/applications/eclipse/plugins 
But no idea if it also goes to the domino\data

**Comment by kbmsg** [comment]
e

**Comment by digihernandez** [comment]
This has been fixed and updated. Thanks for your feedback.
